bin-src/lib/getInstalledStorybookPackages.ts

```diff
@@ -0,0 +1,35 @@
+import execa from 'execa';
+import { viewLayers } from './viewLayers';
+import { supportedAddons } from './supportedAddons';
+
+export const getInstalledStorybookInfo = async () => {
+  const { all } = await execa.command('yarn list --pattern @storybook/ --json', {
+    env: {},
+    timeout: 10000,
+    all: true,
+    shell: true,
+  });
+
+  const { data } = JSON.parse(all as any);
+  const trees: { name: string; version: string }[] = data.trees.map((p: any) => {
+    const [, name, version] = p.name.match(/(.+)@(.+)/);
```

Comment:
This needs a fallback in case the regex matches nothing, otherwise you'll be destructuring `undefined` which is a runtime error. E.g.:
```js
const [, name, version] = p.name.match(/(.+)@(.+)/) || [];
```